Sources/SwiftDocC/Model/Rendering/Variants/VariantOverrides.swift

```diff
@@ -28,7 +28,7 @@ public class VariantOverrides: Codable {
     
     /// Initializes a value given overrides.
     public init(values: [VariantOverride] = []) {
-        self.values = values
+        add(contentsOf: values)
```

Comment:
Any specific reason for this change?

Reply:
I'm not sure I ended up actually using this (so I can likely revert if we don't want to take this now), but I think I would expect that that VariantOverrides field is set up accurately when initialized with an array like this.

So I would expect it properly collects any duplicate traits and append them to a `patch` in the way the `add` function already works.

Otherwise calling `init` can give you an invalid VariantOverrides model.